msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -109,28 +109,33 @@
 					format: charRemFormat
 				 });
 				 updateGradeAssignment();
-			 });				 
+
+                if($('#revise\\:createForumsForGroups\\:1').prop('checked') == true) {
+                    setAutoCreatePanel($('#revise\\:createForumsForGroups'));
```

Comment:
setAutoCreatePanel doesn't seem to even use the jquery object passed in. You could perhaps just remove that from the function signature and remove these selectors in the calls. I don't even know why it's called setAutoCreatePanel. It is forums, though :)

Reply:
Yeah, can be removed.